tutorials/mnist_tutorial_pure_tf.py

```diff
@@ -31,32 +32,56 @@
 """
 
 
-class MLP(object):
+class MLP(Model):
   """
   An example of a bare bones multilayer perceptron (MLP) class.
   """
 
   def __init__(self, layers, input_shape):
+    super(MLP, self).__init__()
+
+    self.layer_names = []
     self.layers = layers
     self.input_shape = input_shape
-    for layer in self.layers:
+    for i,layer in enumerate(self.layers):
```

Comment:
This looks like a PEP8 error, because there's no whitespace after the ",". Is the PEP8 check failing to run on this file?